[2021-03-07 19:25:29,097] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: get_from_clickhouse_1.save 2021-03-07T19:25:27.610456+00:00 [queued]>
[2021-03-07 19:25:29,108] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: get_from_clickhouse_1.save 2021-03-07T19:25:27.610456+00:00 [queued]>
[2021-03-07 19:25:29,109] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-07 19:25:29,109] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-07 19:25:29,109] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-07 19:25:29,117] {taskinstance.py:1063} INFO - Executing <Task(_PythonDecoratedOperator): save> on 2021-03-07T19:25:27.610456+00:00
[2021-03-07 19:25:29,121] {standard_task_runner.py:52} INFO - Started process 49 to run task
[2021-03-07 19:25:29,124] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'get_from_clickhouse_1', 'save', '2021-03-07T19:25:27.610456+00:00', '--job-id', '169', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/get_from_clickhouse_1/main.py', '--cfg-path', '/tmp/tmpe430kl58', '--error-file', '/tmp/tmps8w9w78a']
[2021-03-07 19:25:29,124] {standard_task_runner.py:77} INFO - Job 169: Subtask save
[2021-03-07 19:25:29,158] {logging_mixin.py:104} INFO - Running <TaskInstance: get_from_clickhouse_1.save 2021-03-07T19:25:27.610456+00:00 [running]> on host 020f66ce59e6
[2021-03-07 19:25:29,192] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=iotoi
AIRFLOW_CTX_DAG_ID=get_from_clickhouse_1
AIRFLOW_CTX_TASK_ID=save
AIRFLOW_CTX_EXECUTION_DATE=2021-03-07T19:25:27.610456+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-07T19:25:27.610456+00:00
[2021-03-07 19:25:29,281] {logging_mixin.py:104} INFO - None
[2021-03-07 19:25:29,289] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=get_from_clickhouse_1, task_id=save, execution_date=20210307T192527, start_date=20210307T192529, end_date=20210307T192529
[2021-03-07 19:25:29,314] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-07 19:25:29,336] {local_task_job.py:146} INFO - Task exited with return code 0
